matplotlibcpp.h

```diff
@@ -6,11 +6,16 @@
 #include <stdexcept>
 #include <iostream>
 
-#if __cplusplus > 199711L
+#if __cplusplus > 199711L || _MSC_VER > 1800
 #include <functional>
 #endif
 
-#include <python2.7/Python.h>
+#include <Python.h>
```

Comment:
Can you change the logic such that it uses `python2.7/Python.h` by default but with an option to specify another path? I would like to keep the number of required additional compiler flags to a minimum, and it would probably break their working setup for a lot of people using this.